nss-day-cohort-41/nutshell-asking-for-roses
# Display message to the DOM

# Description
User messages are now rendering to the DOM. In the process, I refactored the dashboard loading code so it happens in sequence with loading the pertinent user data.

There is currently an issue when a new user is registered, and then logs off, the registration form is still displayed.

Fixes # (issue)
Session storage now clears upon Logoff

## Type of change
Please delete options that are not relevant.
# Testing Instructions
Serve the webpage, then login or create a new user. All messages are displayed within the message window. If logging in as an existing user, Edit buttons will appear next to that user's messages.

# Checklist:
